app/components/api-token-row.js

```diff
@@ -8,7 +8,7 @@ export default Component.extend({
 
   didInsertElement() {
     let input = this.element.querySelector('input');
-    if (input.focus) {
+    if (input && input.focus) {
```

Comment:
Great catch on this! I wish I had seen this before this weekend; someone reported revoking tokens wasn't working so I reverted a commit because I didn't know what the fix was right away. [I'm glad you came up with the same fix I eventually did](https://github.com/rust-lang/crates.io/pull/1923)!

Reply:
Nice. Once your PR is merged, the current merge conflict should resolve itself. Thanks.